templates/app-template-11ty/_includes/layouts/base.njk

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="/static/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="Web site created using create-snowpack-app" />
+    <link rel="stylesheet" type="text/css" href="/static/index.css" />
+    <title>Snowpack App</title>
+  </head>
+  <body>  
+    <noscript>You need to enable JavaScrifddpt to run this app.</noscript>
```

Comment:
```suggestion
    <noscript>You need to enable JavaScript to run this app.</noscript>
```

Reply:
Given that Eleventy is pretty pro server-first should the template even have this in it? Most Eleventy sites work without JavaScript.

Reply:
I can't stop thinking about how `JavaScrifddpt` got in there...

Reply:
this response on Twitter answers it well: https://twitter.com/FredKSchott/status/1259596071891636225

basically, JS is the main thing that you'd use this template for. But, you're obviously welcome to remove this in your app.